### DISCUSSION TITLE
Reply to message through /mentions

### DISCUSSION THREAD

**Discussion by notheowner** [Chatterino/chatterino2#4012] [Ideas]
I don't know how useful or practical this could be for people, but I noticed the reply to message button is present in the /mentions tab but has no functionality in said tab.
Maybe when you right click reply someone in the /mentions tab it could bring up the reply field as it would in a normal chat tab.
![U9IN1Mj](https://user-images.githubusercontent.com/86596679/192289925-7f31125d-7990-472b-b6ab-dca6d7619350.png)

**Comment by RyanPotat** [comment]
I was just about to suggest this as well, I feel it would be a great feature.
